.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Run '...'
+2. See error
+\
+If possible, please provide the task package you are using.
```

Comment:
```suggestion
If possible, please provide a simplified version of the task package you are using,
but *don't* send any code that should be kept non-public, such as solutions to tasks that haven't been used yet.
```

Reply:
Any reason for resolving this conversation? In case of not applying suggestions, it's good to leave a message about why you're against.

Reply:
I thought I committed this change

Reply:
No problem. I see that you usually write the commit hash before resolving the conversation (which I appreciate, it helps!), it's a nice method of not resolving without a commit.